ipywidgets/widgets.py

```diff
@@ -4,16 +4,17 @@
 
 class StaticWidget(object):
     """Base Class for Static Widgets"""
-    def __init__(self, name=None, divclass=None):
+    def __init__(self, name=None, description=None, divclass=None):
```

Comment:
Similar to the description optional argument in the ipywidgets library. Workaround to avoid overriding user-defined names for the widgets.